README.md

```diff
@@ -68,6 +68,10 @@ Run the following to generate HTML files and run the web site locally.
 bundle exec jekyll serve
 ```
 
+## Deployment
+
+On commits to the `master` branch of `apache/arrow-site`, the rendered static site will be published to the `asf-site` branch using GitHub Actions. On a fork, it will deploy to your `gh-pages` branch for deployment via GitHub Pages; this is useful for previewing changes you're proposing. To enable this deployment on your fork, you'll need to sign up for GitHub Actions [here](https://github.com/features/actions/signup).
```

Comment:
The current GitHub Actions configuration does the followings:

  * push to master: always push to `asf-site` (`apache/arrow-site` or forked repository isn't related)
  * create a pull request: always push to `gh-pages` of forked repository (*)

(*) But this isn't work for pull request on apache/arrow-site for now. This works for pull request on forked repository if we enable GitHub Actions on forked repository. e.g. https://github.com/kou/arrow-site/pull/4

Reply:
I'll revise the readme. IMO push to master on a fork should build and push to gh-pages, not asf-site. But PR instead works, assuming it works. Though it's not seeming to work on my fork and I can't figure out why. I don't see any other places where I could turn on GitHub Actions https://github.com/nealrichardson/arrow-site/pull/3

Reply:
OK. I've changed behavior on "push to master":

  * apache/arrow-site: push to asf-site on apache/arrow-site
  * forked repositories: push to gh-pages on forked repository